src/panels/developer-tools/event/event-subscribe-card.ts

```diff
@@ -22,6 +22,8 @@ class EventSubscribeCard extends LitElement {
     event: HassEvent;
   }> = [];
 
+  @state() private _error: string | null = null;
```

Comment:
Can we use `undefined` instead of `null` to align with the rest of the code base